app/src/main/java/com/deniscerri/ytdl/util/UiUtil.kt

```diff
@@ -1560,10 +1560,8 @@ object UiUtil {
                 }
             }else{
                 cut.alpha = 0.3f
-                if (!invalidDuration) {
-                    cut.setOnClickListener {
-                        cutDisabledClicked()
-                    }
+                cut.setOnClickListener {
```

Comment:
One thing here. If duration is invalid like -1, then cut should never be allowed to be clicked, as u can see in previous code. Can you code this in aswell pls? Thnx

Reply:
but if its invalid why not just show the same "cut is not available for this item" snack instead?

Reply:
Well you coded it to only trigger that for 0:00, so it will just tell the user to update data which would do nothing it would still bring -1

Reply:
True. What I meant is, why not include `-1` in the if else there (and show "... not available"), instead of blocking it here (like it is now). I am fine with either option, however.

Reply:
Ah, if that is the case, then sure go ahead

Reply:
done